src/components/BookStatus/BookRecordsUsers/UserRecordsMobileModal/UserRecordsMobileModal.css

```diff
@@ -0,0 +1,14 @@
+.records-dialog.MuiDialog-root .MuiDialog-container {
+  align-items: flex-end;
+  background-color: rgba(247, 247, 249, 0.2);
+}
+
+.records-dialog .MuiPaper-root.MuiDialog-paper {
+  width: 100%;
+  max-width: 100%;
```

Comment:
Max-width may not be needed here since we have the width attribute set to the same value.